content/pt-br/docs/tasks/administer-cluster/kubelet-credential-provider.md

```diff
@@ -0,0 +1,147 @@
+# de todos os provedores serão devolvidos ao kubelet. Se vários provedores forem chamados
+# para uma única imagem, os resultados são combinados. Se os provedores retornarem 
+# chaves de autenticação sobrepostas, o valor do provedor anterior da lista é usado.
+providers:
+   # name é o nome necessário do provedor de credenciais. Deve corresponder ao nome do
+   # executável do provedor visto pelo kubelet. O executável deve estar no 
+   # diretório bin do kubelet (definido pelo sinalizador --image-credential-provider-bin-dir).
+   - name: ecr
+     # matchImages é uma lista obrigatória de strings usadas para corresponder às imagens para
+     # determinar se este provedor deve ser invocado. Se uma das strings corresponder à
+     # imagem solicitada do kubelet, o plug-in será invocado e terá uma chance
+     # para fornecer credenciais. Espera-se que as imagens contenham o domínio de registro
+     # e caminho da URL.
+     #
+     # Cada entrada em matchImages é um padrão que pode opcionalmente conter uma porta e um caminho.
+     # Globs podem ser usados no domínio, mas não na porta ou no caminho. Globs são suportados
+     # como subdomínios como '*.k8s.io' ou 'k8s.*.io' e domínios de nível superior como 'k8s.*'.
+     # A correspondência de subdomínios parciais como 'app*.k8s.io' também é suportada. Cada glob só pode corresponder
+     # a um único segmento de subdomínio, então `*.io` **não** corresponde a `*.k8s.io`.
+     #
+     # Existe uma correspondência entre uma imagem e uma matchImage quando todas as opções abaixo são verdadeiras:
+     # - Ambos contêm o mesmo número de partes de domínio e cada parte faz correspondência.
+     # - O caminho do URL de um matchImages deve ser um prefixo do caminho do URL da imagem de destino.
+     # - Se matchImages contiver uma porta, a porta também deverá corresponder à imagem.
+     #
+     # Valores de exemplo de matchImages:
+     # - 123456789.dkr.ecr.us-east-1.amazonaws.com
+     # - *.azurecr.io
+     # - gcr.io
+     # - *.*.registry.io
+     # - Registry.io:8080/path
+     matchImages:
+       - "*.dkr.ecr.*.amazonaws.com"
+       - "*.dkr.ecr.*.amazonaws.cn"
+       - "*.dkr.ecr-fips.*.amazonaws.com"
+       - "*.dkr.ecr.us-iso-east-1.c2s.ic.gov"
+       - "*.dkr.ecr.us-isob-east-1.sc2s.sgov.gov"
+     # defaultCacheDuration é a duração padrão em que o plug-in armazenará as credenciais na memória
+     # se a duração do cache não for fornecida na resposta do plug-in. Este campo é obrigatório.
+     defaultCacheDuration: "12h"
+     # Versão de entrada necessária do exec CredentialProviderRequest. O CredentialProviderResponse retornado
+     # DEVE usar a mesma versão de codificação da entrada. Os valores atualmente suportados são:
+     # - credentialprovider.kubelet.k8s.io/v1
+     apiVersion: credentialprovider.kubelet.k8s.io/v1
+     # Argumentos para passar ao comando quando for executá-lo.
+     # +opcional
```

Comment:
`+optional` is a Go comment tag, so it should not be translated. See https://github.com/kubernetes/community/blob/8b47d07727de4004b78394d45afd84e03ac9d7b4/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required.